packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -56,7 +56,7 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    * Whether the ComboBox auto completes. As the user is inputing text, it will be suggested potential matches from the list of options. If
    * the combo box is expanded, this will also scroll to the suggested option, and give it a selected style. Defaults to false.
    */
-  autoComplete?: string;
+  autoComplete?: boolean;

```

Comment:
We should not be making this change. It was changed in the past specifically to align with the html attribute. See: https://github.com/OfficeDev/office-ui-fabric-react/commit/21c744979934285a8759d93ce8cd8748a16053a5

Reply:
Ok, thanks for the explanation. Will abandon